### PR TITLE
b11.0.0 patch

### PR DESCRIPTION
**Pull request by alantreadway**
Patches for:
- [[Charts] Tooltip not appearing in some scenarios in non-Chrome browsers](https://ag-grid.atlassian.net/browse/AG-13523)
- [[Charts] Interactivity - Focus indicator doesn't always show when pressing Shift + TAB](https://ag-grid.atlassian.net/browse/AG-13643)
- [[Charts] Allow unique ids for DOM elements across multiple chart instances](https://ag-grid.atlassian.net/browse/AG-13708)
- [[Charts] Regression - Left/Right arrows don't move cursor when editing annotation](https://ag-grid.atlassian.net/browse/AG-13715)

### CODE REVIEW

**Comment by olegat** [comment]
LGTM.

There is a merge conflict in `packages/ag-charts-website/e2e/keyboard-nav.spec.ts`, yet none of the changes in `packages/ag-charts-website/e2e/` are critical. We can drop the added tests if it facilitates the merge.

**Review by olegat — APPROVED** [review]
LGTM.

There is a merge conflict in `packages/ag-charts-website/e2e/keyboard-nav.spec.ts`, yet none of the changes in `packages/ag-charts-website/e2e/` are critical. We can drop the added tests if it facilitates the merge.

File: packages/ag-charts-community/src/scale/bandScale.ts [hunk 1]
```diff
@@ -224,8 +224,11 @@ export class BandScale<D, I = number> implements Scale<D, number, I> {
     }
 
     protected ordinalRange(i: number) {
-        const { _inset: inset, _step: step } = this;
-        return inset + step * i;
+        const { _inset: inset, _step: step, range } = this;
+        const min = Math.min(range[0], range[1]);
+        const max = Math.max(range[0], range[1]);
+        // Clamp to account for FP issues
+        return clamp(min, inset + step * i, max);
```
Comment (olegat) [inline packages/ag-charts-community/src/scale/bandScale.ts:231]: > [!WARNING]
> I cannot confirm nor deny whether this change is correct. @jacobp100 Can you please verify this change?

The rest of the LGTM

Comment (alantreadway) [reply packages/ag-charts-community/src/scale/bandScale.ts:231]: This is part of the expected patch for Jacob's fix.

**Pull request merged**